app/src/main/java/org/wikipedia/navtab/NavTab.kt

```diff
@@ -36,23 +37,45 @@ enum class NavTab constructor(
             return SuggestedEditsTasksFragment.newInstance()
         }
     },
+    CONTRIBUTE(R.string.nav_item_contribute, R.id.nav_tab_contribute, R.drawable.selector_nav_edits) {
```

Comment:
Can we add the `if` statement for the string and keep using the same `EDITS` enum so that we don't have to worry about possible crashes in `MainFragment`?

```
EDITS(if (ContributionsDashboardHelper.contributionsDashboardEnabled) R.string.nav_item_suggested_edits else R.string.nav_tab_contribute, R.id.nav_tab_edits, R.drawable.selector_nav_edits) {
        override fun newInstance(): Fragment {
            return SuggestedEditsTasksFragment.newInstance()
        }
    }
```

Reply:
done.